### ISSUE TITLE
Music and video files cannot be played.

### ISSUE THREAD

**Issue by shom13** [zhanghai/MaterialFiles#324]
Firstly I really love your app. So desperately trying to use it. But since installation: 
- a few of my sd card files got corrupted that has never happened before

- music files wont play in my music player or vlc

- messes up my music playlists due to some reason.

**Comment by zhanghai** [comment]
- What is your device model and system version?
- What files got corrupted? Are they the ones that are copied/moved with this app, or random files not touched by this app?
- I'm using OnePlus 5 and Android 9.0, but my music (.mp3 or .flac) can play normally in VLC and music players after being opened from this app.
- Where is your music playlist? Is it in the system music player, or a third party music player?

**Comment by shom13** [comment]
- Its installed on moto c plus, on android 7.
- File systems that got corrupted are   a .pdf file, one .opml file and a folder of mp3 files(all files), the common string is, they were all on the sd card where i was trying out the ftp with nautilus file manager to check if its working. So moved a few files but cannot say exactly which ones. 
- Cannot play the music files, or video files using the file manager. Using other file managers, namely amaze or stock I can.
- Playlist is created through a 3rd party app called music player, altgough if i install any other app for example play music gomusic or timber, the same playlist would appear.

**Comment by zhanghai** [comment]
Did this happen on an internal SD card (the one available directly when the app first opens), or an external SD card (the one added via "Add external storage")?

**Comment by shom13** [comment]
Its happening on an external device, an sd card. I dont have much data on it, just my music files and a few other files here and there. 

I understand it must be difficult figuring out the problem without being able to reproduce it. Feel free to ask anything, as i genuinely wish to continue using the app, with your help ofcourse

**Comment by shom13** [comment]
As of now, i will clear cache, revert permissions, uninstall and reinstall the application. I will report back if I notice anything that might help you figuring it out.

**Comment by shom13** [comment]
After reinstall, it hasnt done anything like it did before, but unfortunately still wont play my music files, some files play irregularly, but most wont.

Under files(at the top), it keeps saying "loading..." most of the times, only sometimes it would give information of files on it.

**Comment by dhammel** [comment]
hm, im having an issue opening audio files through vlc. A single click may or may not open it, rarely does. A double click most times works, which is odd. Sometimes needs multiple clicks, if it works i plays just fine through vlc. Issue doesn't occur with file managers like solid explorer.

Im on a pixel 4, android 11 October update, rooted. Version 1.1.2 of Material Files.

**Comment by zhanghai** [comment]
Could you re-test this on v1.1.4? I did fix some related things, not sure if they would be the culprit for this isse though.

**Comment by dhammel** [comment]
Tested on 1.1.4 (fdroid version) issue still occurs, but seems to be a lot more reliable though, single clicks work better. Every double click though seems to open it through vlc now. So whatever you did, improoved upon it just not completely fixed though.

**Comment by gdnzz** [comment]
Agreed, it would be nice if the app could play music and video files.

**Comment by kevinhwang91** [comment]
> Agreed, it would be nice if the app could play music and video files. @zhanghai

Yes, that's why I can't leave `cx file explorer` which isn't open source. 

For some reason, I have a lot of audio in my ftp server and play them online on every device.
